shell/common/node_bindings.cc

```diff
@@ -151,6 +152,23 @@ void V8FatalErrorCallback(const char* location, const char* message) {
   *zero = 0;
 }
 
+bool AllowWasmCodeGenerationCallback(v8::Local<v8::Context> context,
+                                     v8::Local<v8::String>) {
+  // If we're running with contextIsolation enabled, fall back to Blink's logic.
+  // We switch on the embedder data field count here because if we're in a ctx
+  // that Node.js doesn't control, the number of embedder data fields will
+  // always be less than the min index Node.js uses.
+  v8::Isolate* isolate = context->GetIsolate();
+  if (node::Environment::GetCurrent(isolate) == nullptr) {
+    return blink::V8Initializer::WasmCodeGenerationCheckCallbackInMainThread(
```

Comment:
Should we also _only_ do this in the renderer? I don't see a valid case for this to be called from the browser.  I think if we end up in this block in the browser process we should return false

Reply:
Should do a runtime process check